### PR TITLE
prefer CVH role

### PR DESCRIPTION
**Pull request by cainlevy**
This functionality depends on CVH v5.3.0. I've tested integration against CMob and Pathways and the changes look manageable.

### CODE REVIEW

File: src/RoomClient.ts [hunk 1]
```diff
@@ -188,7 +185,7 @@ class RoomClient {
         videoDisabled,
       };
       this.emitter.emit("onConnectionState", this.connectionState);
-      if (this.broadcastConnectionState)
+      if (this.role === "monitor")
```
Comment (cainlevy) [inline src/RoomClient.ts:188]: @jackofsometrade thoughts?

**Comment by cainlevy** [comment]
@jackofsometrade this is blocking the heartbeat changes. i'm especially interested in any feedback you have about the `broadcastConnectionState` change i tagged above.

File: src/RoomClient.ts (hunk 1 above)
Comment (jackofsometrade) [reply src/RoomClient.ts:188]: This seems fine to me, although wouldn't we want this to be `!==`?

Comment (cainlevy) [reply src/RoomClient.ts:188]: ack yes thank you

**Comment by cainlevy** [comment]
@jackofsometrade thanks for catching that!

**Comment by cainlevy** [comment]
bumping. i've been stuck on this step since yesterday afternoon. not sure how to make progress.

**Comment by cainlevy** [comment]
added some rudimentary test coverage for the feature i nearly broke. i'm going to merge this and continue on.

**Pull request merged**